### ISSUE TITLE
Reducing multiple png files to a pdf generate blank pdf

### ISSUE THREAD

**Issue by ipfaze** [mothsART/machin#9]
### What happened?

When trying to reduce multiple png file to a pdf file, the command below produces a blank pdf with multiple pages.
```bash
ls ~/Downloads/*.png | ~/Projects/Rust/machin/target/debug/machreduce -o result.pdf
```

### What did you expect to happen?

We expect to generate a pdf file containing all the images.

### Machin version

I'm using the current version available on the master branch, the last commit hash is [2c33926edda9f7471331f90aa9b4d499bea4aaf6](https://github.com/mothsART/machin/commit/2c33926edda9f7471331f90aa9b4d499bea4aaf6)

### Steps to reproduce it

1. Pull the master branch
2. Build project using `cargo build`
3. Download some png image
4. Try to reduce it using `ls *.png | machreduce -o result.pdf`

The result of those steps should produce a blank pdf with multiple pages.

**Comment by mothsART** [comment]
Probably the same problem than https://github.com/mothsART/machin/issues/5

**Comment by mothsART** [comment]
@ipfaze : Can you confirm that https://github.com/mothsART/machin/pull/13 solves also this issue ?

**Comment by ipfaze** [comment]
Hi @mothsART 

#13 fixes this issue and also #5.

Thank you so much for your work !
